cypress/fixtures/mockPostDetails.json

```diff
@@ -0,0 +1,59 @@
+{
+  "data": {
```

Comment:
Check out the structure of the post data coming back from graphQL, I think this is "posts" instead of "data", before testing we want to be sure that these signatures match completely